### ISSUE TITLE
[CAIA A11y Support]: Design System Team, Design System Date component 

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/va.gov-team#57845]
### What do you need support with?

The DST got an experimental design request for the date component. 
https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1711

Here is some information from the experimental design request
Usage
Va-date will be disabled based on conditional logic supplied to the component.

Behavior
Inputs will disable when condition supplied is met.

Accessibility
From what we know this is the lesser of two evils that our design team has chosen. Per the reasoning on buttons (https://design.va.gov/components/button/#do-not-disable-buttons) we know that we shouldn't disable and opt to remove; however, we believe that removing/re-adding the date field in the middle of the page will be jarring to screenreader users.

### When do you need this support by?

5/15/2023

### How urgent is this request?

Not urgent, we can use this help when CAIA has the time and resources

### Supporting artifacts (if any)

- [Design System Product outline](https://vfs.atlassian.net/l/cp/0da50yhH)
- [Date input component](https://design.va.gov/components/form/date-input)
- [[Experimental Design [va-date]](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1711#top)](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1711)

...

### Who's on your team?

Team name: Design System
OCTO-DE product owner: Matthew Dingee
Product manager: Carol Wong
Designer: Barb Denney and Lily Boot
FE engineer: Jami Gibbs 
Product/team Slack channel: #platform-design-system

### Do you have an accessibility champion on your team?

No, we don't have a champion (yet)

**Comment by HeatherRienks** [comment]
Following 
+ @OptionSelect  for vis

**Comment by sara-amanda** [comment]
@caw310  looking through this ticket and the additional tickets linked here, and[ I found the UXPin link. Can you confirm this the most-current prototype ](https://preview.uxpin.com/f43733391f52c409c4d3152e9a4b101a392e29df#/pages/161460736/simulate/no-panels)we should be looking at? Many thanks!

**Comment by HeatherRienks** [comment]
@caw310 - I will confirm with the team :)

cc: @sara-amanda

**Comment by sara-amanda** [comment]
Thank you @HeatherRienks !

**Comment by sara-amanda** [comment]
## CAIA Recommendation
@caw310 @HeatherRienks @coforma-terry 

An option here is to follow the standard of grouping with one set of questions per page. This keeps it simple with one "item" per page. So, something you could try here is [breaking apart this page into two different steps - detailed below.](https://preview.uxpin.com/f43733391f52c409c4d3152e9a4b101a392e29df#/pages/161460736/simulate/no-panels).

### Step 2
   - Type of work (dropdown) 
   - Employer name (text field)
   - Do they currently work here?  (checkbox)
### Step 3
- Date Input(s)
   - **If they check the box in step 2**, that greyed out option does not need to show at all. 
   - **If they _do not_ check the box in step 2**, then that field will be available for input. 
### Resource Links
For more information on one thing per page, these are excellent resources to check out: 
- [Original smashing article](https://www.smashingmagazine.com/2017/05/better-form-design-one-thing-per-page/)
- [gov uk](https://www.gov.uk/service-manual/design/form-structure)
- [va design principle](https://design.va.gov/patterns/help-users-to/complete-a-sub-task#design-principles)

### Image of Current Step 2

![Screenshot 2023-05-08 at 3.08.50 PM.png](https://images.zenhubusercontent.com/63dbe59975ee0eef93e952b2/0b994a78-8969-4712-bc4a-ffed5606c149)

#### Reference Image: Current Step 2 & Image Description

- **Heading**: Add a job
- **Paragraph**: Tell us about any jobs you’ve had in the past 2 years that you received pay stubs for. You’ll need to provide your income information if it’s a current job.
##### Required Fields
- Type of work (dropdown)
- Employer Name (text field)
- Date you started work at this job    
  - Month (Dropdown) and Year (number field)
- Date you stopped work at this job 
  - Month (Dropdown) and Year (number field)
- Checkbox: I currently work here. 
    - When checked date you stopped at this job is greyed out.
